libvips/mosaicing/mosaic1.c

```diff
@@ -114,6 +114,8 @@ vips__lrmerge1( VipsImage *ref, VipsImage *sec, VipsImage *out,
 	VipsBuf buf;
 	char text[1024];
 
+	t[0] = vips_image_new();
```

Comment:
o.O this hasn't worked for a while! It's clearly not very important haha.

Reply:
Indeed, I guess I assumed this was covered by the existing `test_mosaicing.py` tests, but that was clearly not the case.